docs/manual/manual_dronecan.rst

```diff
@@ -0,0 +1,77 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_dronecan:
+
+***********************************************
+DroneCAN
+***********************************************
+
+DroneCAN (previously known as UAVCAN or UAVCANv0), is an open protocol for communication over a CAN bus. DroneCAN is supported by both PX4

```

Comment:
open source protocol?

Reply:
This language is actually taken straight from the DroneCAN documentation page: https://dronecan.github.io/

They call themselves an "open protocol", so I used the same terminology